### PR TITLE
Use the .NET 6.0 SDK for building (Arcade7)

### PR DESCRIPTION
**Pull request by JoeRobich**
Test Build: https://dev.azure.com/dnceng/internal/_build/results?buildId=1460011&view=results (microsoft)

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
     "xcopy-msbuild": "16.10.0-preview2"
   },
   "msbuild-sdks": {
-    "Microsoft.DotNet.Arcade.Sdk": "7.0.0-beta.21474.2",
-    "Microsoft.DotNet.Helix.Sdk": "7.0.0-beta.21474.2"
+    "Microsoft.DotNet.Arcade.Sdk": "7.0.0-beta.21566.10",
+    "Microsoft.DotNet.Helix.Sdk": "7.0.0-beta.21566.10"
   }
-}
+}
```
Comment (sharwell) [inline global.json:18]: ⚠️ This should not have changed. If a tool created this change, please file a bug report for that tool (changing the line ending at the end of the file is not allowed).

**Review by sharwell — APPROVED** [review]

File: global.json (hunk 1 above)
Comment (JoeRobich) [reply global.json:18]: Looks like it happened when I resolved conflicts locally. Should we update our .editorconfig to specify what we want for .json files.

Comment (JoeRobich) [reply global.json:18]:
```suggestion
}

```

**Pull request merged**